packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -121,9 +122,28 @@ class ApplicationCommand extends Base {
       this.options ??= [];
     }
 
+    if ('default_member_permissions' in data) {
+      /**
+       * The default bitfield used to determine whether this command be used in a guild
+       * @type {?Readonly<PermissionsBitField>}
+       */
+      this.defaultMemberPermissions = data.default_member_permissions
+        ? new PermissionsBitField(BigInt(data.default_member_permissions)).freeze()
+        : null;
+    }
```

Comment:
Maybe do something like this?

```suggestion
    } else {
    	this.defaultMemberPermissions ??= null;
    }
```